src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,15 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        if (value == 0) {
+            return "0";
+        }
+        StringBuilder sb = new StringBuilder();
+        while (value > 0) {
+            int isBinary = value % 2;
```

Comment:
you don't need separate variable "isBinary"
try to remove it

Reply:
done